app/lib/three_scale/semantic_form_builder.rb

```diff
@@ -31,7 +31,10 @@ def text_field_with_errors( method, opts = {})
     # Get a column object for a specified attribute method - if possible.
     #
     def column_for(method) #:nodoc:
-      @object.column_for_attribute(method) if @object.respond_to?(:column_for_attribute) && @object.try(:has_attribute?, method)
+      # our old formtastic expects nil as in Rails 4, not a null object
+      # https://github.com/rails/rails/pull/15878
+      column = super
+      ActiveRecord::ConnectionAdapters::NullColumn === column ? nil : column
```

Comment:
Would be good to have a test so when we upgrade formtastic it will break and then make the appropriate changes here ;)

Reply:
see `#test_column_for_override_needed`